lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -812,6 +812,11 @@ public void checkCanGrantExecuteFunctionPrivilege(SystemSecurityContext context,
     {
     }
 
+    @Override
+    public void checkCanGrantExecuteFunctionPrivilege(SystemSecurityContext context, FunctionKind functionKind, CatalogSchemaRoutineName functionName, TrinoPrincipal grantee, boolean grantOption)
+    {
+    }
```

Comment:
I guess it should contain 
```
        if (functionKind == TABLE) {
            denyGrantExecuteFunction(functionName.toString());
        }
```

Reply:
I checked locally and in theory you never will end up here because File-based Access Control disallow to create view with table function in a first place.
But to be 100% correct in situation when user creates some views and then switch to File-based Access Control this additional check is required.

Reply:
i agree we should deny here, just like we deny TABLE functions in checkCanExecuteFunction.